src/graphic/minimap_renderer.cc

```diff
@@ -45,11 +45,30 @@ int round_up_to_nearest_even(int number) {
 	return number % 2 == 0 ? number : number + 1;
 }
 
+uint16_t check_vision(const Widelands::Coords& coords,
+					  const Widelands::Player* player,
+					  const Widelands::EditorGameBase& egbase) {
+	const Widelands::Map& map = egbase.map();
+	const int32_t mapwidth = map.get_width();
+	Widelands::FCoords f = map.get_fcoords(coords);
+	Widelands::MapIndex i = Widelands::Map::get_index(f, mapwidth);
+	move_r(mapwidth, f, i);
```

Comment:
Please add a comment explaining what this does

Reply:
I copy-pasted this line from `do_draw_minimap()`, and testing shows it's wrong here. Removing...